src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -1510,6 +1514,34 @@ private Index ResetIndexInternal(Index index, bool start = true)
             }
         }
 
+        private Index ResetIndexSideBySideInternal(Index index)
+        {
+            try
+            {
+                var definitionClone = new IndexDefinition();
+                var definition = index.GetIndexDefinition();
+                definition.CopyTo(definitionClone);
+
+                definitionClone.ForceUpdate = true;
```

Comment:
Do you really need to have `ForceUpdate`? Cannot you just pass it as parameter to `HandleStaticIndexChange()`?

Reply:
Changed